src/devices/Qmc5883l/Status.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Iot.Device.Qmc5883l
+{
+    /// <summary>
+    /// Status flags for QMC5883L first register
+    /// </summary>
+    [Flags]
+    public enum Status : byte
```

Comment:
It doesn't seem to be needed to be public. I guess internal is enough in this case as if I read correctly, you're not using it outside of the class